docs/_includes/themes/zeppelin/_navigation.html

```diff
@@ -27,6 +27,11 @@
                 <li><a href="{{BASE_PATH}}/quickstart/tutorial.html">Tutorial</a></li>
                 <li role="separator" class="divider"></li>
                 <li class="title"><span><b>Basic Feature Guide</b><span></li>
+                <!-- li><span><b>Tutorial</b><span></li -->
+                <li><a href="{{BASE_PATH}}/tutorial/tutorial.html">Tutorial</a></li>
+                <li><a hrev="{{BASE_PATH}}/tutorial/installing_with_flink_and_spark_cluster.html">Install Zeppelin with Flink and Spark Clusters Tutorial</a></li>
```

Comment:
How about put this docs below [this line](https://github.com/apache/zeppelin/blob/master/docs/_includes/themes/zeppelin/_navigation.html#L35)?